### DISCUSSION TITLE
Runout Sensor

### DISCUSSION THREAD

**Discussion by mallison01** [xblax/flashforge_ad5m_klipper_mod#78] [General]
My runout sensor does not seem to be working. I rebooted into stock to make sure it works and it does. but in the mod it does not.

I looked and did not see any discussion or issues to this?

**Comment by johncotten** [comment]
It has been closed. #50 Also you can see my mod at the bottom that is to replace with one that detects motion as well, for like a nozzle clog or filament tied.

**Reply by mallison01** [reply]
sorry, its late and i didn't even think to look at the closed issues.

I will add this to my config and see if it works then.

Thanks.

**Comment by ryryonline** [comment]
I was testing this out and noticed that if you start a print without filament loaded it will try to print. To test, you have to put filament into the feed tube and pull it out for it to activate. Is there a way to trigger a detection at the beginning?
